src/components/interfaces/MOBILE_API.xml

```diff
@@ -3094,11 +3094,7 @@
             <description>The short friendly name of the climate control module.
                 It should not be used to identify a module by mobile application.</description>
         </param>
-        <param name="currentTemperatureAvailable" type="Boolean" mandatory="false">
-            <description>
-                Availability of the reading of current temperature.
-                True: Available, False: Not Available, Not present: Not Available.
-            </description>
+        <param name="currentTemperatureAvailable" type="Boolean" mandatory="false" since="5.0">
```

Comment:
Why was description removed?

Reply:
@jacobkeeler According to your first comment in https://github.com/smartdevicelink/sdl_core/pull/2207#pullrequestreview-146501436

Reply:
I was referring to the parameter as a whole, not the description